test/web-platform-tests/to-run.yaml

```diff
@@ -72,6 +72,56 @@ simple-requests.htm: [timeout, Maybe https://github.com/jsdom/jsdom/issues/1833
 
 ---
 
+DIR: css/css-color
+
+animation/**: [fail, Unknown]
+color-mix-currentcolor-visited-getcomputedstyle.html: [fail, Unknown]
+color-mix-missing-components.html: [fail, Unknown]
+inheritance.html: [fail, Unknown]
+light-dark-basic.html: [fail, Depends on matchMedia]
+nested-color-mix-with-currentcolor.html: [fail, Unknown]
+parsing/color-computed-*.html: [fail, Unknown]
+parsing/color-computed.html: [fail, Unknown]
+parsing/color-invalid-color-function.html: [fail, Unknown]
+parsing/color-invalid-color-layers-function.html: [fail, Unknown]
+parsing/color-invalid-lab.html: [fail, Unknown]
+parsing/color-invalid-named-color.html: [fail, Unknown]
+parsing/color-invalid-relative-color.html: [fail, Unknown]
+parsing/color-mix-out-of-gamut.html: [fail, Unknown]
+parsing/color-valid-color-function.html: [fail, Unknown]
+parsing/color-valid-color-layers-function.html: [fail, Unknown]
+parsing/color-valid-color-mix-function.html: [fail, Unknown]
+parsing/color-valid-contrast-color-function.html: [fail, Unknown]
+parsing/color-valid-hsl.html: [fail, Unknown]
+parsing/color-valid-hwb.html: [fail, Unknown]
+parsing/color-valid-lab.html: [fail, Unknown]
+parsing/color-valid-relative-color.html: [fail, Unknown]
+parsing/color-valid-rgb.html: [fail, Unknown]
+parsing/color-valid-system-color.html: [fail, Unknown]
+parsing/opacity-computed.html: [fail, Unknown]
+parsing/opacity-valid.html: [fail, Unknown]
+parsing/relative-color-out-of-gamut.html: [fail, Unknown]
```

Comment:
It's a bit surprising that we fail these given all the work you put into the color parser!